apps/www/components/layout/user-account-nav.tsx

```diff
@@ -96,7 +94,7 @@ export function UserAccountNav({ user }: UserAccountNavProps) {
       className="px-3"
       variant="default"
       size="sm"
-      onClick={signInModal.onOpen}
+      onClick={() => router.push("/signin")}
```

Comment:
So for this code we are not using the modal that opens, but right into the /signin page?

Reply:
This seems like a better option to have a single page that handles sign-in rather than having a modal for it

Currently, the modal and the sign-in page have two different forms for login, which seemed a bit confusing.

![image](https://github.com/projectx-codehagen/Badget/assets/98402835/eacfbdd3-a1d2-4fb7-a160-db897e41d6e8)
![image](https://github.com/projectx-codehagen/Badget/assets/98402835/1f78a2a3-e58f-4f7f-a570-fe6200ea12d6)